wlc/__init__.py

```diff
@@ -195,6 +202,28 @@ def list_languages(self):
         """List languages in the instance."""
         return self.list_factory("languages/", Language)
 
+    def create_project(
+        self, name, slug, website, source_language_name=None, source_language_code=None
+    ):
+        """Create a new project in the instance."""
+        data = {
+            "name": name,
+            "slug": slug,
+            "web": website,
+        }
+        if source_language_name and source_language_code:
+            data["source_language"] = {
+                "name": source_language_name,
+                "code": source_language_code,
+            }
+
+        return self.post("projects/", **data)
+
+    def create_component(self, project, **kwargs):
```

Comment:
Are there any required `kwargs`? If yes perhaps we should be explicit about what's required and what isn't (and do some error checking) ?

Reply:
I was offloading the validations to the API since that's how the library is mostly constructed. I can check for required parameters from the documentation and API trials and include the validations.